### DISCUSSION TITLE
is it possible to get and locate the specify line number

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [wojtekmaj/react-pdf#1617] [Q&A] [answered]
when the next time reload the same pdf, it always show the first page. when the pdf scroll, is it possible to get the exactly line number and located the line when the next time load the pdf?

**Comment by wojtekmaj** [comment, marked as answer]
In general, it's up to the browser to maintain scroll position, but given the delay between the page load and PDF load, this may not work correctly. My advice would be to use IntersectionObserver to track which page is currently displayed on the screen, keep it in sync with URL e.g. by adding #page-2 to it, and upon refresh, reading this hash to programatically move the user back to where they were.
